src/tools/schema2base.ts

```diff
@@ -58,6 +57,12 @@ export abstract class Schema2Base {
     if (manifest.errors.some(e => e.severity !== 'warning')) {
       return;
     }
+
+    this.namespace = manifest.meta.namespace;
+    if (!this.namespace) {
+      throw new Error(`Arcs manifest '${src}' does not define a code generation namespace`);
```

Comment:
nit: 
`Arcs manifest '${src}' does not define a namespace; required for code generation`

or `Namespace is required in '${src}' for code generation`.

Reply:
Updated!